ui/src/app/shared/components/tri-state-button.component.ts

```diff
@@ -63,6 +63,7 @@ export class TriStateButtonComponent implements AfterViewInit, DoCheck {
       this.button.nativeElement.disabled = true;
     } else {
       this.label = `Unregister all ${this._items.length} selected ${appOrApps}`;
+      this.button.nativeElement.disabled = false;
```

Comment:
probably set the this.button.nativeElement.disabled to false on line 57 and remove the one from here and from line 60.   Will do on merge.